src/Microsoft.NetCore.Analyzers/Core/Security/SystemSecurityCryptographyResources.resx

```diff
@@ -261,6 +261,15 @@
   <data name="SetViewStateUserKeyMessage" xml:space="preserve">
     <value>The subclass {0} of Page does not set the ViewStateUserKey property in method OnInit</value>
   </data>
+  <data name="UseSharedAccessProtocolHttpsOnly" xml:space="preserve">
+    <value>Use SharedAccessProtocol HttpsOnly</value>
+  </data>
+  <data name="UseSharedAccessProtocolHttpsOnlyDescription" xml:space="preserve">
+    <value>Https is identical to Http because it follows the same basic protocols, but Https was developed to allow authorization and secured transactions, which is more secure and more private than Http. So, it is recommended to use Https.</value>
```

Comment:
>Https is identical to Http because it follows the same basic protocols, but Https was developed to allow authorization and secured transactions, which is more secure and more private than Http. So, it is recommended to use Https [](start = 11, length = 228)

HTTPS encrypts network traffic.  Use HttpsOnly, rather than HttpOrHttps, to ensure network traffic is always encrypted to help prevent disclosure of sensitive data.